app/assets/javascripts/exponential_backoff.js.coffee

```diff
@@ -0,0 +1,33 @@
+@organizations = @organizations || {};
+retry = (url, callback, configObject) ->
+  configObject.retries++
+  setTimeout(
+    () ->
+      configObject.retryIn *= 1.25
+      organizations.exponentialBackoff(url, callback, configObject)
+  , configObject.retryIn
+  )
+
+setDefaults = (configObject) ->
+  configObject = configObject || {}
+  configObject.retries = configObject.retries || 0
+  configObject.retryIn = configObject.retryIn || 2000
```

Comment:
This code was reviewed by sebas balay in this pr or another project https://github.com/Wolox/projects-manager/pull/5